Cargo.toml

```diff
@@ -7,4 +7,4 @@ edition = "2018"
 crate-type = ["cdylib"]
 
 [dependencies]
-redis-module = { version="0.5.0", features = ["experimental-api"]}
+redis-module = { git="https://github.com/RedisLabsModules/redismodule-rs.git", branch="hash_get_set", features = ["experimental-api"]}
```

Comment:
Is the branch setting here ok?

Reply:
now that the redis-module 0.6.0 was released I can switch it to 0.6.0